Makefile

```diff
@@ -8,5 +8,6 @@ clean:
 	true
 
 install:
+	rm -r $(DST)
```

Comment:
Předpokládám, že jde jen o kontrolu -- ujistit se, že v `$(DST)` nic není, že?

Reply:
Když jsem to debuggoval, tak jsem narazil na problém, kdy jsem si to pomocí `sudo make install` nainstaloval a zjistil jsem, že po založení repa tam stále existují složky `out` a `results`. Ty tam byly právě kvůli tomu, že se nesmazali v `$(DST)`, tedy ano, je to kontrola toho, že tam není nic jiného, než jen to, co tam má být.